README.md

```diff
@@ -5,7 +5,8 @@
 <div align="right">
   <sup>
     <a href="#the-challenge">English</a> ·
-    <a href="#お題">日本語</a>
+    <a href="#お題">日本語</a> .
+    <a href="#tantangan">Indonesia</a> .
```

Comment:
The result text should be like this `English · 日本語 · Indonesia` instead of this `English · 日本語 . Indonesia .`
This symbol ` · ` is different with this ` . `